### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token '<'

### DISCUSSION THREAD

**Discussion by arnhazra** [vercel/community#1576] [Help]
### Summary

After each commit to github and after deployment when I open the application I am getting "Uncaught SyntaxError: Unexpected token '<'"

Application URL: https://lenstack.vercel.app/
Repository URL: https://github.com/arnhazra/lenstack

Tech Stack: Express JS & React (Served from Express)

### Example

https://lenstack.vercel.app/

**Comment by amyegan** [comment]
Hi Arnab! It looks like you already solved the problem. Are you still getting this error?

**Reply by arnhazra** [reply]
Not yet, using a trick to force refresh on load
